new-charts/tc-bullet-chart.scss

```diff
@@ -34,3 +34,21 @@
   @include bullet-chart__serie($sentiment, $color, sentiment);
 }
 @include bullet-chart__serie(value, $tc-color--serie-2);
+
+@mixin colorBulletAttrByName($attr, $serie, $color, $el: $serie) {
+  .bullet-chart__bullet[data-#{$attr}='#{$serie}'] .bullet__#{$attr},
```

Comment:
I tried to have the lightest mixin, hope it isn't too factorized. Tell me if we need to simplified by spliting in three different mixin (and removing el element ) ?